docs/_pages/rtm_client.md

```diff
@@ -165,24 +170,28 @@ web.channels.list()
 
 ### Listening for message subtypes
 
-There's a shortcut syntax for listening to specific message subtypes, just format the event type as
+There's a shortcut syntax for listening to specific [message
+subtypes](https://api.slack.com/events/message#message_subtypes), just format the event type as
 `message::${message_subtype}`. See the example below.
 
 ```javascript
+const { RTMClient } = require('@slack/client');
+// SNIP: the initialization code shown above is skipped for brevity
+
 rtm.on(`message::channel_name`, (event) => {
   // For structure of `event`, see https://api.slack.com/events/message/channel_name
   console.log(`A channel was renamed from ${message.old_name} to ${message.name}`);
 });
 ```
 
-If you rename a channel in which the bot is a member, you should see the log message.
+With this program running, when you rename a channel in which the bot is a member, you should see the log message.
```

Comment:
This sentence is confusing. Maybe "When the program is running and you rename a channel your bot is a member of, you should see the logged message."